controllers/admin.js

```diff
@@ -26,20 +10,8 @@ const getAdmins = async (_req, res) => {
 
 const getAdmin = async (req, res) => {
   const userId = req.params.userId
-  const admin = await User.findOne({
-    _id: userId,
-    role: ADMIN
-  }).lean()
-
-  if (!admin) throw createError(404, USER_NOT_REGISTERED)
-
-  const adminResponse = {
-    id: admin._id,
-    firstName: admin.firstName,
-    lastName: admin.lastName,
-    email: admin.email
-  }
 
+  const adminResponse = await adminService.getAdmin(userId)
   res.status(200).json({
```

Comment:
you don't need to set `status(200)` . it's value by default

Reply:
check all cases in this pr

Reply:
Defaults are not always explicit, so I use the codes that need to be used. A developer who doesn’t know it, may start questioning what the status code actually is. So as I said, I’ll remain explicit here